features/facebookImport/src/model/analyses/ministories/about-pictures-data-analysis.js

```diff
@@ -44,7 +43,6 @@ export default class AboutPicturesDataAnalysis extends RootAnalysis {
     async analyze({ zipFile, dataAccount }) {
         const pictureEntries = await this._userPicturesFromExport(zipFile);
         if (pictureEntries.length >= PICTURES_THRESHOLD)
-            dataAccount.analyses[analysisKeys.picturesCount] =
-                pictureEntries.length;
+            dataAccount.processedData._picturesCount = pictureEntries.length;
```

Comment:
This seems like it's breaking with the old model of having _analysis results_ - now there's just a soup of _processed data_? What if there's another analysis that wants to record something called `_picturesCount`? I strongly prefer the previous design.

Reply:
This would've been an issue with the old model too. The only difference now, is we are putting things into processedData (rather than analysisData because we hold report info as well)  and doing it using a class with attributes rather than an enum.